native/src/components/PoiFiltersModal.tsx

```diff
@@ -0,0 +1,158 @@
+import React, { ReactElement } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Button } from 'react-native-elements'
+import { useTheme } from 'styled-components'
+import styled from 'styled-components/native'
+
+import { PoiCategoryModel, PoiModel } from 'api-client'
+
+import { ClockIcon, MobilityIcon } from '../assets'
+import Modal from './Modal'
+import SettingsSwitch from './SettingsSwitch'
+import Text from './base/Text'
+
+const Container = styled.View`
+  flex: 1;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0 16px;
+`
+
+const SubTitle = styled(Text)`
+  font-size: 14px;
+  color: ${props => props.theme.colors.textColor};
+  font-family: ${props => props.theme.fonts.native.decorativeFontBold};
+`
+
+const Section = styled.View`
+  padding-top: 16px;
+  width: 100%;
+`
+
+const Row = styled.View`
+  flex-direction: row;
+  padding-top: 16px;
+  width: 100%;
+`
+
+const StyledText = styled.Text`
+  font-size: 14px;
+  color: ${props => props.theme.colors.textColor};
+  font-family: ${props => props.theme.fonts.native.decorativeFontRegular};
+  padding: 4px;
+`
+
+const SortHint = styled.Text`
+  align-self: flex-end;
+  font-size: 12px;
+  color: ${props => props.theme.colors.textColor};
+  font-family: ${props => props.theme.fonts.native.decorativeFontRegular};
+  padding: 0 4px;
+`
+
+const FlexEnd = styled.View`
+  flex: 1;
+`
+
+const TileRow = styled(Row)`
+  justify-content: space-between;
+  align-content: space-between;
+  flex-wrap: wrap;
+`
+
+const PoiCategoryTile = styled.Pressable<{ active: boolean }>`
+  background-color: ${props => (props.active ? props.theme.colors.themeColor : props.theme.colors.backgroundColor)};
+  padding: 8px;
+  align-items: center;
+  width: 100px;
+  height: 80px;
+  border-radius: 18px;
+  elevation: 5;
+  shadow-color: ${props => props.theme.colors.textColor};
+  shadow-offset: 0px 1px;
+  shadow-opacity: 0.2;
```

Comment:
I find the tiles barely visible, maybe we can go up to 0.4?

Reply:
This is an iOS only issue so no idea how it actually looks on iOS. However, so far we only have opacities ~0.2 with a max of 0.25. I think for more we should check back with a designer. I would anyway reuse the feedback buttons from #971 once done.